ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -205,8 +205,10 @@ def yield_dashboard_lineage(
         """
         Yields lineage if config is enabled
         """
-        if self.source_config.dbServiceName:
-            yield from self.yield_dashboard_lineage_details(dashboard_details) or []
+        for db_service_name in self.source_config.dbServiceNames:
```

Comment:
@codingwithabhi this will break if `self.source_config.dbServiceNames` comes `None`, since it is not a required field in json schema